package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -1804,4 +1820,28 @@ def generate_table(self, output_format=None):
         logger.debug(
             "WBridge: Stored results as table with %(num_records)d entries.",
             vars())
-        self.table = table
+        self.results.table = table
+
+    @property
+    def table(self):
+        wmsg = ("The `table` attribute was deprecated in MDAnalysis 2.0.0 "
+                "and will be removed in MDAnalysis 3.0.0. Please use "
+                "`results.table` instead")
+        warnings.warn(wmsg, DeprecationWarning)
+        return self.results.table
+
+    @property
+    def _network(self):
```

Comment:
I think we should just remove _network as it is a private variable and is not used outside the module.

Reply:
Ok, that makes it even easier.

Reply:
As mentioned above it was used a lot in the tests, but this PR will change that and therefore it is definitely better to just remove it.